botorch/models/approximate_gp.py

```diff
@@ -351,13 +362,32 @@ def __init__(
             inducing_point_allocator: The `InducingPointAllocator` used to
                 initialize the inducing point locations. If omitted,
                 uses `GreedyVarianceReduction`.
+            outcome_transform: An outcome transform that is applied to the training
+                data during instantiation and to the posterior during inference.
+                NOTE: If this model is trained in minibatches, an outcome transform
+                with learnable parameters (such as `Standardize`) would update its
+                parameters for each minibatch, which is undesirable. If you do intend
+                to train in minibatches, we recommend you not use an outcome transform
+                and instead pre-transform your whole data set before fitting the model.
```

Comment:
Thanks for adding these docstrings!

Good to be explicit about defaults now that many models have `Standardize` by default.
```suggestion
            outcome_transform: An outcome transform that is applied to the training
                data during instantiation and to the posterior during inference. By 
                default, no outcome transform is used. It is not recommended to use 
                an outcome transform if this model is trained in minibatches, because 
                an outcome transform
                with learnable parameters (such as `Standardize`) would update its
                parameters for each minibatch, which is undesirable. If you do intend
                to train in minibatches, we recommend you not use an outcome transform
                and instead pre-transform your whole data set before fitting the model.
```